dashboard/src/components/js/Button/Button.test.js

```diff
@@ -25,7 +25,7 @@ describe(Button, () => {
 
   describe("Themes", () => {
     Object.keys(ButtonThemes).map(k => {
-      it(`apply the ${k} theme`, () => {
+      return it(`apply the ${k} theme`, () => {
```

Comment:
rather than returning I would use `forEach` rather than `map` but I guess it's the same.